### PR TITLE
feat: added telegram integration

### PR DESCRIPTION
**Pull request by Kenny-Gin1**
- Added a new Telegram integration under "Internal Integrations" as well. Please check it out: https://denoted.sentry.io/settings/developer-settings/telegram-6b9651/

Not tested yet as it needs to be deployed.

### CODE REVIEW

File: apps/web/sentry.server.config.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 // https://docs.sentry.io/platforms/javascript/guides/nextjs/
 
 import * as Sentry from '@sentry/nextjs';
+import { TelegramTransport } from "./src/lib/telegram/telegramTransport";
```
Comment (philipandersson) [inline apps/web/sentry.server.config.js:6]: 👀

File: apps/web/src/pages/api/sentry/webhook.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+const botToken = process.env.TELEGRAM_API_KEY;
+const chatId = process.env.TELEGRAM_CHAT_ID;
+const bot = new TelegramBot(botToken as string, { polling: false });
+
+const sentryWebhookHandler = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  if (req.method === 'POST') {
```
Comment (philipandersson) [inline apps/web/src/pages/api/sentry/webhook.ts:12]: We need to verify signature so we know the payload hasn't been tampered with.
It acts auth: https://docs.sentry.io/product/integrations/integration-platform/webhooks/#verifying-the-signature

**Review by philipandersson — CHANGES_REQUESTED** [review]

**Review by philipandersson — APPROVED** [review]

File: apps/web/src/pages/api/sentry/webhook.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      await bot.sendMessage(chatId as string, errorMessage);
+    }
+
+    res.status(200).json({ message: "Webhook processed successfully." });
+  } else {
+    res.status(401).json({ message: "Unauthorized or method not allowed." });
+  }
+};
+
+export const config = {
```
Comment (Kenny-Gin1) [inline apps/web/src/pages/api/sentry/webhook.ts:70]: @philipandersson Vad gör denna?

Comment (philipandersson) [reply apps/web/src/pages/api/sentry/webhook.ts:79]: Disables parsing of the body so we can verify the signature properly.

**Pull request merged**